src/cljs/rems/administration/create_catalogue_item.cljs

```diff
@@ -15,10 +16,20 @@
             [rems.util :refer [navigate! fetch post! put! trim-when-string]]))
 
 (defn- update-loading [db]
-  (let [progress (::loading-progress db)]
-    (if (<= 2 progress)
-      (dissoc db ::loading-progress ::loading?)
-      (assoc db ::loading-progress (inc progress)))))
+  (let [progress (::loading-progress db)
+        catalogue-item-id (::catalogue-item-id db)]
+    (cond
+      (<= progress 1)
+      {:db (assoc db ::loading-progress (inc progress))}
+
+      (nil? catalogue-item-id)
```

Comment:
what's this case?

Reply:
If we end up on the page by creating a new catalogue item, instead of editing an existing one, it is indicated by missing catalogue-item-id. Maybe it should be more explicit, I'll think about it a bit.